libcudacxx/include/cuda/std/__cccl/compiler.h

```diff
@@ -20,10 +20,14 @@
 #  define _CCCL_COMPILER_NVHPC
 #elif defined(__clang__)
 #  define _CCCL_COMPILER_CLANG
+#  define _CCCL_CLANG_VERSION (__clang_major__ * 10000 + __clang_minor__ * 100 + __clang_patchlevel__)
 #elif defined(__GNUC__)
 #  define _CCCL_COMPILER_GCC
+#  define _CCCL_GCC_VERSION (__GNUC__ * 10000 + __GNUC_MINOR__ * 100 + __GNUC_PATCHLEVEL__)
 #elif defined(_MSC_VER)
 #  define _CCCL_COMPILER_MSVC
+#  define _CCCL_MSVC_VERSION      _MSC_VER
+#  define _CCCL_MSVC_VERSION_FULL _MSC_FULL_VER
 #elif defined(__IBMCPP__)
```

Comment:
We don't support PowerPC/ppc64le/IBM platform anymore

Reply:
Also, Intel ICC is deprecated

Reply:
Then we should probably emity a warning when a user attempts tp compile for such a platform. Can you please create an issue to track that? Also, please include a reference where we state that support is no longer provided. Thx!

Reply:
I believe that is just coming from libc++ not the toolkit